src/uploadNotionImagesToCloudinary.ts

```diff
@@ -13,12 +13,13 @@ export default async function uploadNotionImagesToCloudinary({
   cloudinaryUrl = process.env.CLOUDINARY_URL || "",
   cloudinaryUploadFolder = process.env.CLOUDINARY_UPLOAD_FOLDER || "",
   logLevel = process.env.NODE_ENV === "development" ? "debug" : "error",
+  uploadExternalsNotOnCloudinary = false,
```

Comment:
Can you make this param also be able to come from an env var? So we continue continue having the same support for passing parameters via env

```suggestion
  uploadExternalsNotOnCloudinary = process.env.UPLOAD_EXTERNALS_NOT_ON_CLOUDINARY
    ? process.env.UPLOAD_EXTERNALS_NOT_ON_CLOUDINARY === "1"
    : false,
```

Maybe it'd be something like this, since a 1 would come as "1"